### PR TITLE
Feature/disable save button

### PR DESCRIPTION
**Pull request by jennadilullo8**
**Add disable functionality**

- Driver: Jenna
- Navigator: Alex, Liliana

- We're able to look at the save button and if both inputs weren't filled out, then the color of the save button would be lighter then what it originally is and the functionality of the idea cards being saved would be disabled.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -34,6 +35,14 @@ function addNewIdea() {
   showNewIdea();
 }
 
+function enableSaveButton() {
```
Comment (lilianaweimer) [inline src/main.js:38]: This could probably be refactored later, but I'm glad we got it working! :)

**Review by lilianaweimer — APPROVED** [review]
Looks great!

File: src/main.js [hunk 2]
```diff
@@ -14,7 +14,8 @@ var ideaArray = [];
 //event listeners
 hamburgerMenu.addEventListener('click', showFilterStarred);
 saveButton.addEventListener('click', addNewIdea);
-
+titleInput.addEventListener('keyup', enableSaveButton);
```
Comment (asodemann18) [inline src/main.js:17]: Nice use of keyup!

**Review by asodemann18 — COMMENTED** [review]
Looks good! Will probably need some refactoring later in the week.

**Pull request merged**